frontend/index.html

```diff
@@ -83,40 +83,7 @@
 			rel="stylesheet"
 		/>
 		<title>highlight.io</title>
-		<script>
-			// We pre-filled your app ID in the widget URL: 'https://widget.intercom.io/widget/gm6369ty'
-			;(function () {
-				var w = window
-				var ic = w.Intercom
-				if (typeof ic === 'function') {
-					ic('reattach_activator')
-					ic('update', w.intercomSettings)
-				} else {
-					var d = document
-					var i = function () {
-						i.c(arguments)
-					}
-					i.q = []
-					i.c = function (args) {
-						i.q.push(args)
-					}
-					w.Intercom = i
-					var l = function () {
-						var s = d.createElement('script')
-						s.type = 'text/javascript'
-						s.async = true
-						s.src = 'https://widget.intercom.io/widget/gm6369ty'
-						var x = d.getElementsByTagName('script')[0]
-						x.parentNode.insertBefore(s, x)
-					}
-					if (w.attachEvent) {
-						w.attachEvent('onload', l)
-					} else {
-						w.addEventListener('load', l, false)
-					}
-				}
-			})()
-		</script>
+		<script type="module" src="./src/intercom.js" defer></script>
```

Comment:
Moved intercom and canny scripts out of html and into their own scripts in src folder because we can't add arbitrary HTML like this in Reflame, but we can add additional entry point scripts. A side benefit is this should result in a slightly smaller document payload, and slightly less network traffic overall since these scripts will stay cached on repeat visits as they rarely ever change. (UPDATE: just found out that this not true for Vite, since all of these scripts just end up getting bundled into a single index.js file...)

Also added defer to keep sequential execution order while keeping network requests parallelized.